machine_learning_hep/processerdhadrons_jet.py

```diff
@@ -645,21 +718,22 @@ def effcorr_response(self, df_noeffcorr):
                 eff = heff_pr.GetBinContent(ipt+1)
                 # limit the prompt(nonprompt) df with current bin ranges and
                 # assign corresponding efficiency to each bin as a new df column
-                df_tmp = seldf_singlevar(df_noeffcorr, "pt_cand", \
-                        self.lpt_finbinmin[ipt], self.lpt_finbinmax[ipt])
+                df_tmp = seldf_singlevar(df_noeffcorr, "pt_cand",
+                                         self.lpt_finbinmin[ipt], self.lpt_finbinmax[ipt])
                 df_tmp['eff'] = eff
-                if eff > 0:
-                    df_tmp['weight'] = 1/eff
+                df_tmp['unity'] = 1.0
```

Comment:
Is this needed?

Reply:
i think it is removed now

Reply:
Fixed in https://github.com/nzardosh/MachineLearningHEP/pull/1